tests/cases/user_test.py

```diff
@@ -856,3 +856,19 @@ def testPrivateUser(self):
             path='/file/chunk', user=pvt, fields=fields, files=files)
         self.assertStatusOk(resp)
         self.assertEqual(resp.json['itemId'], itemId)
+
+    def testNumberUser(self):
+        """
+        Make sure display number of user is correct
+        """
+        # Create an admin user
+        self.model('user').createUser(
+            firstName='Admin', lastName='Admin', login='admin',
+            email='admin@admin.com', password='adminadmin')
+        # Create an user1 and user2
+        self.model('user').createUser('user1', 'passwd1', 'tst1', 'usr1', 'user1@user.com')
+        self.model('user').createUser('user2', 'passwd2', 'tst2', 'usr2', 'user2@user.com')
+        # Make sure we get a 200-level when trying to count users 
```

Comment:
You can probably remove this comment line because it is implied by the name `assertStatusOk`, and also it is implied from the fact that this is a test for that specific endpoint.

Generally try to avoid adding comments for explaining exactly what the code is doing and is obvious from the code.  Instead the comments should communicate intent, or else explain assumptions or what the code is doing that may be complex or non-obvious.  Of course what is "obvious" to one person isn't to another, and it is always a bit a judgement call as to whether it is useful to add in a comment like this.